pyanaconda/ntp.py

```diff
@@ -106,7 +110,12 @@ def ntp_server_working(server_hostname):
     client = ntplib.NTPClient()
```

Comment:
I think we might move this in the else branch as the `client` variable is now used only there ?